### PR TITLE
adds IODLV2 Licence

### PR DESCRIPTION
**Pull request by chespinoza**
adds IODLV2 Licence used by Torino open data

### CODE REVIEW

File: data_license.go [hunk 1]
```diff
@@ -328,6 +331,21 @@ var (
...
 		},
+
+		IODLV2URL: DataLicense{
+			Name:         "Italian Open Data License v2.0",
+			URL:          "https://www.dati.gov.it/content/italian-open-data-license-v20",
+			LegalCodeURL: "https://www.dati.gov.it/content/italian-open-data-license-v20",
+			Permits: []string{
+				Reproduction,
+				Distribution,
+				DerivativeWorks,
```
Comment (dotmancando) [inline data_license.go]: UK's OGL we had `Sharing` in the Permits array. I suppose this one is similar. I'll let @Popcore read the license page and compare this the CC ontology: https://creativecommons.org/ns and then he will decide

Comment (Popcore) [reply data_license.go]: @chespinoza and @dotmancando according to their definition (and the wikipedia one as well) the Italian Open Data License v2.0 is compatible with CC3.0 and [ODBL](https://opendatacommons.org/licenses/odbl/)

Comment (dotmancando) [reply data_license.go]: our definition of OBDL has `Sharing` in Permits array. Shall we add `Sharing` to this license as well?

Comment (Popcore) [reply data_license.go]: I agree with @dotmancando we should have `Sharing` in the `Permits` slice since it is part of the license definition.

Comment (chespinoza) [reply data_license.go]: @dotmancando @Popcore done.

**Pull request merged**